README.md

```diff
@@ -65,89 +121,176 @@ label_maker = cp.LabelMaker(
     window_size="1h",
 )
 ```
+
 ### Search Labels
-Next, we automatically search and extract the labels by using [`LabelMaker.search`](http://docs.compose.ml/en/latest/generated/methods/composeml.LabelMaker.search.html#composeml.LabelMaker.search).
+Next, we automatically search and extract the labels by using [`LabelMaker.search`](https://compose.featurelabs.com/en/latest/generated/methods/composeml.LabelMaker.search.html#composeml.LabelMaker.search). The data must be sorted by the time index before passing into the label maker. For more details on how the label maker works, see [Main Concepts](https://compose.featurelabs.com/en/latest/main_concepts.html).
 
 ```python
 labels = label_maker.search(
-    df,
-    minimum_data="1h",
-    num_examples_per_instance=25,
+    df.sort_values('transaction_time'),
+    num_examples_per_instance=-1,
     gap=1,
     verbose=True,
 )
+
 labels.head()
 ```
-```
-          customer_id         cutoff_time  total_spent
-label_id
-0                   1 2014-01-01 04:13:51        65.11
-1                   1 2014-01-03 15:41:34       101.08
-2                   1 2014-01-05 11:46:10        16.78
-3                   1 2014-01-06 09:54:58       108.16
-4                   1 2014-01-08 08:54:02        48.33
-```
+<table border="0" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th>customer_id</th>
+      <th>cutoff_time</th>
+      <th>total_spent</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>1</td>
+      <td>2014-01-01 00:45:30</td>
+      <td>914.73</td>
+    </tr>
+    <tr>
+      <td>1</td>
+      <td>2014-01-01 00:46:35</td>
+      <td>806.62</td>
+    </tr>
+    <tr>
+      <td>1</td>
+      <td>2014-01-01 00:47:40</td>
+      <td>694.09</td>
+    </tr>
+    <tr>
+      <td>1</td>
+      <td>2014-01-01 00:52:00</td>
+      <td>687.80</td>
+    </tr>
+    <tr>
+      <td>1</td>
+      <td>2014-01-01 00:53:05</td>
+      <td>656.43</td>
+    </tr>
+  </tbody>
+</table>
 
 ### Transform Labels
-With the generated [`LabelTimes`](https://docs.compose.ml/en/latest/generated/composeml.LabelTimes.html#composeml.LabelTimes), we will apply specific transforms for our prediction problem. To make the labels binary, a threshold is applied for amounts exceeding $100.
+With the generated [`LabelTimes`](https://compose.featurelabs.com/en/latest/generated/composeml.LabelTimes.html#composeml.LabelTimes), we will apply specific transforms for our prediction problem. To make the labels binary, a threshold is applied for amounts exceeding $300.
```

Comment:
the code below has 100 not 300 like the text says

Reply:
updated